lib/sms.rb

```diff
@@ -0,0 +1,20 @@
+require 'rubygems'
+require 'twilio-ruby'
+
+class SMSConfirmation
+
+  def initialize
+    account_sid = 'AC6e70d6dcc83c097a6e913c3afbd424a3'
+    auth_token = '901c21df552c5cfcf478c47b4aa310e3'
```

Comment:
Don't commit auth tokens to public repos.  You ought to regenerate this in Twilio to be on the safe side.